### ISSUE TITLE
Error: Qualified path resolution failed: we looked for the following paths, but none could be accessed

### ISSUE THREAD

**Issue by bryanjtc** [Niels-IO/next-image-export-optimizer#134]
The published package has some missing files required when running the command `next-image-export-optimizer`.

Everything in the utils file is missing.
It is being required by `optimizeImages.js`
I'm using v1.5.0
```

Error: Qualified path resolution failed: we looked for the following paths, but none could be accessed.

Source path: /home/bryan/projects/website-ayudinga/.yarn/__virtual__/next-image-export-optimizer-virtual-3896c83a86/0/cache/next-image-export-optimizer-npm-1.5.0-73b45856bb-b977a2e598.zip/node_modules/next-image-export-optimizer/dist/utils/defineProgressBar
Not found: /home/bryan/projects/website-ayudinga/.yarn/__virtual__/next-image-export-optimizer-virtual-3896c83a86/0/cache/next-image-export-optimizer-npm-1.5.0-73b45856bb-b977a2e598.zip/node_modules/next-image-export-optimizer/dist/utils/defineProgressBar
Not found: /home/bryan/projects/website-ayudinga/.yarn/__virtual__/next-image-export-optimizer-virtual-3896c83a86/0/cache/next-image-export-optimizer-npm-1.5.0-73b45856bb-b977a2e598.zip/node_modules/next-image-export-optimizer/dist/utils/defineProgressBar.js
Not found: /home/bryan/projects/website-ayudinga/.yarn/__virtual__/next-image-export-optimizer-virtual-3896c83a86/0/cache/next-image-export-optimizer-npm-1.5.0-73b45856bb-b977a2e598.zip/node_modules/next-image-export-optimizer/dist/utils/defineProgressBar.json
Not found: /home/bryan/projects/website-ayudinga/.yarn/__virtual__/next-image-export-optimizer-virtual-3896c83a86/0/cache/next-image-export-optimizer-npm-1.5.0-73b45856bb-b977a2e598.zip/node_modules/next-image-export-optimizer/dist/utils/defineProgressBar.node

Require stack:
- /home/bryan/projects/website-ayudinga/.yarn/__virtual__/next-image-export-optimizer-virtual-3896c83a86/0/cache/next-image-export-optimizer-npm-1.5.0-73b45856bb-b977a2e598.zip/node_modules/next-image-export-optimizer/dist/optimizeImages.js
    at require$$0.Module._resolveFilename (/home/bryan/projects/website-ayudinga/.pnp.cjs:23144:13)
    at require$$0.Module._load (/home/bryan/projects/website-ayudinga/.pnp.cjs:22994:42)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (/home/bryan/projects/website-ayudinga/.yarn/__virtual__/next-image-export-optimizer-virtual-3896c83a86/0/cache/next-image-export-optimizer-npm-1.5.0-73b45856bb-b977a2e598.zip/node_modules/next-image-export-optimizer/dist/optimizeImages.js:4:27)
    at Module._compile (node:internal/modules/cjs/loader:1159:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at require$$0.Module._extensions..js (/home/bryan/projects/website-ayudinga/.pnp.cjs:23188:33)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at require$$0.Module._load (/home/bryan/projects/website-ayudinga/.pnp.cjs:23007:22)

Node.js v18.12.0
```

**Comment by Niels-IO** [comment]
Hi @bryanjtc,

Thanks for the report! I forgot to include the utils folder in the package.json files field after the change to typescript. It should be fixed in version 1.5.1. Let me know if that fixes it.

**Comment by bryanjtc** [comment]
It's fixed.
